.env.local 

```diff
@@ -0,0 +1,3 @@
+VUE_APP_API_CLIENT="server"
+BACKEND_URL="http://localhost:1337"
+# BACKEND_URL="https://mitra-social.herokuapp.com"
```

Comment:
you really want to commit this?

Reply:
Strange, Actually the whole file shouldn't be checked in, in the gitignore it is registered for ignoring it anyway